README.md

```diff
@@ -1,1193 +1,25 @@
-# Ballerina Azure Event Hubs Connector
+Ballerina Azure Event Hubs Connector
+===================
 
 [![Build](https://github.com/ballerina-platform/module-ballerinax-azure.eventhub/workflows/CI/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-azure.eventhub/actions?query=workflow%3ACI)
 [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-azure.eventhub.svg)](https://github.com/ballerina-platform/module-ballerinax-azure.eventhub/commits/master)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-Connects to Microsoft Azure Event Hubs using Ballerina.
-
-# Introduction
-
-## What is Azure Event Hub?
-
 [Azure Event Hubs](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-about) is a fully managed, real-time 
```

Comment:
Let's remove unnecessary spaces.

Reply:
Shall we reword it as "[Azure Event Hubs](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-about) is a fully managed, real-time data ingress service that is highly scalable, secured, open, and reliable. It ingests data (events) from different sources and reliably distributes it between multiple independent systems for processing, storage, and analysis."

Reply:
Resolved and amended the changes. Thank you for the suggestion.